[DOCS-1875] Update baseline and pinned runs docs for max # of pins (#2081)
wandb/docs: models/runs/compare-runs.mdx

In a workspace with many runs, it can be difficult to keep track of your best performers, production models, failed experiments, or important reference points. The W&B App provides features to help organize and compare runs:

> - **Pinned runs**: Pin up to 6 runs to keep them visible in the workspace and at the top of the runs list. If you have a baseline run, you can pin up to 5 runs because the baseline uses a pinning slot.

Reviewer: What is "pinning slot"?

Author: One of the 6 pins or 5 pins + 1 baseline that you can have. Reworded a bit.
```suggestion
- **Pinned runs**: Pin up to 6 runs to keep them visible in the workspace and at the top of the runs list. If you have a baseline run, you can pin up to 5 runs because the baseline is implicitly pinned.
```

1. Navigate to your workspace.
2. In the run selector or runs table, find the run you want to use as your new baseline.

> 3. Click the action `...` menu, then select **Replace baseline**. <Note>If the menu item is disabled, ensure that you have at least one pinning slot available. If necessary, unpin a pinned run by clicking the circular pin icon next to a pinned run.</Note>

Reviewer: ```suggestion
3. Click the action `...` menu, then select **Replace baseline**. <Note>If the menu item is inactive, ensure that you have at least one pinning slot available. If necessary, unpin a pinned run by clicking the circular pin icon next to a pinned run.</Note>
```

Author: Good call, I couldn't come up with this word in the moment for some reason.

4. The new run becomes the baseline, and the previous baseline is automatically pinned so you can find it easily. Optionally, unpin it by clicking its pin icon.

### Remove the baseline designation
To remove the baseline designation:

1. Navigate to your workspace.
2. In the run selector or runs table, find the current baseline run.

> 3. Click the action `...` menu, then select **Remove baseline**. <Note>If the menu item is disabled, ensure that you have at least one pinning slot available. If necessary, unpin a pinned run by clicking the circular pin icon next to a pinned run.</Note>

Reviewer: ```suggestion
3. Click the action `...` menu, then select **Remove baseline**. <Note>If the menu item is inactive, ensure that you have at least one pinning slot available. If necessary, unpin a pinned run by clicking the circular pin icon next to a pinned run.</Note>
```

2. In the run selector or runs table, find the run you want to use as your new baseline.
3. Click the action `...` menu, then select **Replace baseline**.

The new run becomes the baseline. The previous baseline is automatically pinned if a pinning slot is available, so you can find it easily. To ensure the previous baseline is pinned, make sure you have a free pinning slot available before changing the baseline.

> 

Reviewer 2: I don't think you can get into this scenario... a pinning slot with always be available otherwise you can't replace the baseline:

<img width="327" height="208" alt="Screenshot 2026-01-27 at 9 27 35 AM" src="https://github.com/user-attachments/assets/9da6382c-fe2f-4a4d-8968-609543ab4cfd" />

Author: Done

### Remove the baseline designation
To remove the baseline designation:

1. Navigate to your workspace.
2. In the run selector or runs table, find the current baseline run.
3. Click the action `...` menu, then select **Remove baseline**.

> The previous baseline is automatically pinned if a pinning slot is available, so you can find it easily. To ensure the previous baseline is pinned, make sure you have a free pinning slot available before removing the baseline designation.

Reviewer 2: again, a pinning slot should always be available

Author: Done
